### ISSUE TITLE
Queue tweets for optional review

### ISSUE THREAD

**Issue by leggitta** [leggitta/PervyCliffNotes#7]
Human approval won't be required, but Amy wants to be able to view a queue of scheduled tweets and manually reject.

**Comment by cachilders** [comment]
This one seems pretty essential to me, ya boy.

**Comment by leggitta** [comment]
Maybe through a web dashboard? Whatever database we concoct could store queued tweets. Amy, you or I could then reject ones that make no sense.

**Comment by cachilders** [comment]
The potential for added complexity in even the noninteractive tweets (x-over stuff for instance, and the potential for seemingly innocuous combinations of words turning toxic) the dashboard makes sense. It definitely expands the scope though.

**Comment by cachilders** [comment]
This is an interesting, potentially inappropriate solution to a problem we could maybe look like ours. Limited resource outlay if it could be shimmed. https://github.com/mozillach/gh-projects-content-queue

**Comment by sashaskinner** [comment]
I'm happy to code up a simple dashboard web app that allows a user to reject tweets! I'll need to chat with you, Alan, about how to route the contents of the tweets to the dashboard since I'm not clear on how that whole situation is happening.

**Comment by sashaskinner** [comment]
I'll start working on coding up a dashboard that could serve this purpose. Alan, maybe we can chat sometime about how the tweets will be queued?

**Comment by leggitta** [comment]
Yeah. The only thing I would know how to do is host a flask app on an amazon ec2 instance that displays upcoming tweets and allows someone to delete tweets that don't make sense.

Any thoughts @cachilders ?

Also, @cachilders meet @sashaskinner and @sashaskinner meet @cachilders ! You guys have already exchanged emails I think. Sasha just finished up at Hackbright and wants to add smut to her portfolio!

**Comment by leggitta** [comment]
In 390f8e39ba0e1f774ef7288734bb31f46cb652a6, I made a first pass in flask to get you started @sashaskinner. 

After pulling down the changes, you'll have to update the environment using ...
```
conda env update -f environment.yml
```
Then you can run the app locally using
```
export FLASK_APP=app.py
flask run
```
This should display the localhost url. When you navigate to that, it should display the following table (indivudal tweets are randomized).

<img width="547" alt="screen shot 2017-07-19 at 11 10 12 pm" src="https://user-images.githubusercontent.com/1099867/28402990-7e6e40dc-6cd7-11e7-8da3-d52390361ad2.png">

The scheduled tweet database doesn't actually exist yet, but once it does, we'll have to figure out how to have the delete column interact with it.

**Comment by sashaskinner** [comment]
This looks great! Is there anything in particular you'd like me to work on here? I could make an admin login page and gussy it up a bit until the tweet database is up?

**Comment by leggitta** [comment]
yeah, login would be great! I've never done that before and welcome any
advice as to how to actually authenticate. I'd also like a basic UI to
delete unwanted tweets. It can be as gussied up or as plain as you feel
like.
